packages/quantic/force-app/main/default/lwc/quanticSearchBoxSuggestionsList/quanticSearchBoxSuggestionsList.js

```diff
@@ -262,7 +297,7 @@ export default class QuanticSearchBoxSuggestionsList extends LightningElement {
    * Returns the recent queries that start with the query currently typed by the end user.
    */
   getRecentQueriesThatStartWithCurrentQuery() {
-    return (
+    this._recentQueriesThatStartWithCurrentQuery =
```

Comment:
Any null checks maybe that we need to add for this one and the other function?

For example this one relies on `this.query` but it's possible that this gets called **before** `this.query` has been set to a value no?

So you could surround it with `if(this.query?.length > 0)` or something like that?

Reply:
the code already takes that in consideration as we always use `?` before accessing the query, so the function would not fail if query is `null` or `undefined`, it would just return an empty array in that case.